display_list/skia/dl_sk_paint_dispatcher.h

```diff
@@ -37,8 +37,23 @@ class DlSkPaintDispatchHelper : public virtual DlOpReceiver {
   void setMaskFilter(const DlMaskFilter* filter) override;
   void setImageFilter(const DlImageFilter* filter) override;
 
-  const SkPaint& paint() { return paint_; }
-
+  const SkPaint& paint(bool uses_shader = true) {
+    // On the Impeller backend, we will only support dithering of *gradients*,
+    // and it will be enabled by default (without the option to disable it).
+    // Until Skia support is completely removed, we only want to respect the
+    // dither flag for gradients (otherwise it will also apply to, for
+    // example, image ops and image sources, which are not supported in
```

Comment:
Oops, should probably say "not dithered in" otherwise it looks like we don't implement drawImage...